scripts/end-to-end-tests.sh

```diff
@@ -52,10 +52,11 @@ verify_install_all_before_fail () {
     }
 }
 
-verify_github_tool Rojo "rojo-rbx/rojo" "6.0.0"
-verify_github_tool remodel "rojo-rbx/remodel" "0.9.1"
-verify_github_tool stylua "JohnnyMorganz/stylua" "0.11.3"
+verify_github_tool Rojo "rojo-rbx/rojo" "7.3.0"
+verify_github_tool remodel "rojo-rbx/remodel" "0.11.0"
+verify_github_tool stylua "JohnnyMorganz/stylua" "0.18.10
+verify_github_tool lune "filiptibell/lune" "0.6.7"
 
-verify_gitlab_tool darklua "seaofvoices/darklua" "0.7.0"
+verify_gitlab_tool darklua "seaofvoices/darklua" "0.8.0"
 
-verify_install_all_before_fail selene "Kampfkarren/selene" "0.22.0"
+verify_install_all_before_fail selene "Kampfkarren/selene" "0.666.31337"
```

Comment:
This should be left as is; the tool tests that it _does_ get installed after introducing a fake tool. Look at the impl and you'll see what I mean, the idea is that _all_ valid tools install even if an invalid one is in the list.